content/en/docs/Tasks/creating-a-catalog.md

```diff
@@ -1,43 +1,43 @@
 ---
```

Comment:
Should we add a netlify redirect from the old page name to the new page name, so we don't break existing links?